spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -123,6 +133,11 @@ private static File obtainScreenshotDirectory(Context context) throws IllegalAcc
     return dirMethod;
   }
 
+  private static boolean isLollipopOrAbove() {
+    int currentApiVersion = new Integer(Build.VERSION.SDK).intValue();
```

Comment:
At the time I thought Cupcake support might be possible, but I can't retrace my logic on that one.  (Neither why nor how.) Thanks for the fix :)